lib/landrush/action/common.rb

```diff
@@ -71,7 +71,7 @@ def read_machine_hostname
           return machine.config.vm.hostname
         end
 
-        "#{Pathname.pwd.basename}.#{config.tld}"
+        "#{Pathname.pwd.basename}.#{config.tld[0]}"
```

Comment:
>    config.landrush.tld = [ 'one.org', 'two.org' ]
>    config.landrush.host 'one.org'
>    config.landrush.host 'two.org'

That's what I mean, you still need 'config.landrush.host'. Would you not prefer something like:

    config.landrush.host 'two.org' true

Where the boolean specifies whether the host entry is visible on the host. Really all this means is in the case of OS X that there will be an entry 'two.org' in _/etc/resolver_.

You only use _config.landrush.tld_, since atm that is the config option which will generate the required config, right? 

Wildcard subdomains come for "free". What I mean is that:

    config.landrush.tld = 'one.org'
    config.landrush.host 'one.org'
    config.landrush.host 'two.org'

works just fine with subdomains, if you just do the following:

    sudo cp -r /etc/resolver/one.org /etc/resolver/two.org